src/courseware/course/sequence/sequence-navigation/UnitButton.test.jsx

```diff
@@ -79,7 +94,64 @@ describe('Unit Button', () => {
   it('handles the click', () => {
     const onClick = jest.fn();
     render(<UnitButton {...mockData} onClick={onClick} />);
-    fireEvent.click(screen.getByRole('tab'));
+    fireEvent.click(screen.getByRole('tabpanel'));
     expect(onClick).toHaveBeenCalledTimes(1);
   });
+
+  it('focuses bookmark-button after pressing Enter or Space', async () => {
+    jest.useFakeTimers();
+
+    const { container } = render(
+      <>
+        <UnitButton {...mockData} />
+        <button id="bookmark-button" type="button">Bookmark</button>
```

Comment:
[clarify]: I think the bookmark button already exists in the house? Can we find it without creating a custom button with an ID?

Reply:
not exists. This is tests only for UnitButton